USAGE.md

```diff
@@ -327,6 +327,25 @@ Potential spelling errors will be printed in the continuous integration log alon
 Words in `build/assets/custom-dictionary.txt` are ignored during spellchecking.
 Spellchecking is currently only supported for English language manuscripts.
 
+## AI-assisted authoring
+
+The workflow [`ai-revision`](.github/workflows/ai-revision.yaml) is available to assist authors in writing their manuscripts.
+It uses large language models (LLM) to revise the manuscript text, fixing spelling and grammar errors, and improving the sentence structure and the writing style with section-specific prompts.
+It is manually triggered by the user (it never runs automatically), and it generates a pull request with suggested revisions.
+Then the user can review these changes and merge the pull request if they are acceptable.
+More information about this tool is available in [this manuscript](https://greenelab.github.io/manubot-gpt-manuscript/).
+
+You must provide a secret with name `OPENAI_API_KEY` containing your OpenAI API token.
+The easiest way to set this is to go to your organization's settings page and, within "Secrets and variables," select "Actions."
+Next, create a secret with the appropriate name and the value of the API token.
+By default, the tool uses the model `text-davinci-003`.
+Make sure to check the [pricing](https://openai.com/api/pricing/) of the OpenAI API.
+The cost for a revision of a standard manuscript should be around $0.50.
+The workflow allows to specify the branch and file names (in the `content/` directory) to revise, the language model to use, and the output branch name.
+Internally, the workflow uses the tool [Manubot AI Editor](https://github.com/greenelab/manubot-ai-editor) to revise the manuscript.
+For more advanced users, the behavior of the Manubot AI Editor or the parameters used for the language model can be changed using environment variables.
+These variables can be changed in the workflow file (`ai-revision.yaml`).
+
```

Comment:
Per the prior discussion, this is where we could add a sentence or two about recommended practices for writing with AI tools. That could include a suggestion about listing them in the methods but not as co-authors.

Do we also want to add something here or in the "Citing Manubot" subsection about citing the GPT models and the Manubot AI Editor if they used for revision?

Reply:
I've added a few sentences about the controversy of using these tools in scholarly writing.

I leave the decision to you and @dhimmel about the citation.

Reply:
If Manubot expands to become a suite of tools that are described in their on manuscripts, I like the idea of including them in the citation section. It encourages others to contribute their workflows or tools back to rootstock and/or the manubot package like you did.

What do you think @dhimmel?

Reply:
@dhimmel we should make a decision about expanding the citation section before merging.

Reply:
Yeah, I think it makes sense to add the ai revision citation to USAGE.

Reply:
I don't think that we should demand that users of ai-revision subcommand cite the manuscript but can make them aware of the option.

Reply:
Agreed, we don't need to demand citation. At the very end of "Citing Manubot" we can add something like

> To cite the Manubot AI Editor, see `@doi:10.1101/2023.01.21.525030`:
> 
> <reference using Manubot's default CSL>

@miltondp could you please add that?

Reply:
Done!